### PR TITLE
Adding dRonin copyright/author statements

### PR DESCRIPTION
**Pull request by droidicus**
Adding dRonin copyright/author statements for all merged PR's from #1 through #178. One exception is logging.c, since there are multiple outstanding PRs that already have this change.

No functional changes, unless I made a bad bad mistake.

[Review on Reviewable](https://reviewable.io/reviews/d-ronin/dronin/184)

### CODE REVIEW

**Comment by fujin** [comment]
awesome thanks so much for doing this! :+1:

**Comment by mlyle** [comment]
@droidicus This looks great!

Can you do me a big favor though, and pull out all the makefile changes / all the changes in packaging?  We have a lot of PRs open in that area and I'm concerned it will clash with those outstanding PRs.

File: ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright (c) 2010 Google Inc.
+// Copyright (C) 2015 dRonin, http://dronin.org
 // All rights reserved.
 //
 // Redistribution and use in source and binary forms, with or without
```
Comment (mlyle) [inline ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc]: I think these files we imported, but didn't change.  Shouldn't have our copyright.

Comment (mlyle) [inline ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc]: (except the one file in libcrashreporter-qt that i changed)

**Comment by mlyle** [comment]
Everything looks good except that one thing with breakpad/libcrashreporter-qt!

**Comment by mlyle** [comment]
This should sequence in after #181 just to be careful / keep rebases down.

**Comment by droidicus** [comment]
Review status: 0 of 171 files reviewed at latest revision, 1 unresolved discussion.

---

*[ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc, line 5 \[r2\]](https://reviewable.io:443/reviews/d-ronin/dronin/184#-K4y9wDQHECHwfp8LYyE-r2-5:-K4zJZtvMxLYeODpnUH2:1254889376) ([raw file](https://github.com/d-ronin/dronin/blob/698a1b406e6aa9cd8e91e593d0925048b7501076/ground/gcs/src/libs/libcrashreporter-qt/breakpad/client/linux/handler/exception_handler.cc#L5)):*
This file was changed in 8d261ab463b29f3fe8ae0b9ad84c473975327bb7

---


*Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/d-ronin/dronin/184)*

**Pull request merged**